runtime/include/sandbox_set_as_error.h

```diff
@@ -48,14 +49,20 @@ sandbox_set_as_error(struct sandbox *sandbox, sandbox_state_t last_state)
 	/* State Change Bookkeeping */
 	assert(now > sandbox->timestamp_of.last_state_change);
 	sandbox->last_state_duration = now - sandbox->timestamp_of.last_state_change;
+	if (last_state == SANDBOX_RUNNING_SYS)
```

Comment:
If this is the style of sledge, then great!

But I'd never want to use multi-line conditions without explicit `{}`. It opens you up to too many accidental control-flow mistakes.

Reply:
Done.